src/Kentico.Xperience.AiraUnified/Admin/PathValidationHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Kentico.Xperience.AiraUnified.Admin;
+
+internal static class PathValidationHelper
```

Comment:
Just a little detail - classes with extension methods should be called <XY>Extensions

Reply:
https://stackoverflow.com/questions/2700675/c-sharp-naming-convention-for-extension-methods-for-interface